cg/constants/bcl_convert_metrics.py

```diff
@@ -17,6 +18,19 @@ class BclConvertQualityMetricsColumnNames(Enum):
     Q30_BASES_PERCENT: str = "% Q30"
 
 
+SAMPLE_SHEET_HEADER = {
+    BclConverter.BCL2FASTQ: ",".join(
+        SampleSheetNovaSeq6000Sections.Data.COLUMN_NAMES.value[BclConverter.BCL2FASTQ]
+    ),
+    BclConverter.BCLCONVERT: ",".join(
+        SampleSheetNovaSeq6000Sections.Data.COLUMN_NAMES.value[BclConverter.DRAGEN]
+    ),
+    BclConverter.DRAGEN: ",".join(
+        SampleSheetNovaSeq6000Sections.Data.COLUMN_NAMES.value[BclConverter.DRAGEN]
+    ),
+}
```

Comment:
Any specific reason why you didn't add this in `cg/constants/demultiplex.py`?

Reply:
I just followed Henrik's suggestion - where do you think it fits best? @diitaz93

Reply:
Nevermind, I think the place is fine 🙂